advanced_html_css/accessibility/wai_aria.md

```diff
@@ -58,29 +58,29 @@ When using such ARIA attributes, you would provide an `id` to one element, and y
 
 #### `aria-label`
 
-The `aria-label` attribute overrides any native label and modifies the name property in the accessibility tree, though it's best used when an element doesn't already have a native label. When you add `aria-label` to an element, you pass in a string as the value, which will become that elements accessible name. `aria-label` doesn't work on every HTML element, though. Adding the attribute to a plain `<div>` or a `<span>` will have no effect, for example.
+The `aria-label` attribute overrides any native label of an element and modifies its name property in the accessibility tree. It is best used when an element doesn't already have a native label. When added, the string value of `aria-label` attribute  becomes the element's accessible name. **However**, [`aria-label` does not have any effect on some HTML elements](https://github.com/w3c/aria/issues/756), such as `<div>` or a `<span>`. 
 
-A common use for this attribute can be for the "close" button often seen in menus or modals:
+A common use for `aria-label` can be found in the "close" buttons of menus or modals:
 
 ~~~html
 <button type='button' aria-label='Close menu'>X</button>
 ~~~
 
-Instead of a screen reader announcing, "X, button", which wouldn't make any sense to the user, it would announce, "Close menu, button". Another way you could use `aria-label` is on landmark elements (our Semantic HTML lesson gets another shoutout... again!):
+Instead of a screen reader announcing, "X, button", which makes little sense to the user, it would announce, "Close menu, button". Another way you could use `aria-label` is on landmark elements (our Semantic HTML lesson gets another shoutout... again!):
```

Comment:
Let's add a link:

```suggestion
Instead of a screen reader announcing, "X, button", which makes little sense to the user, it would announce, "Close menu, button". Another way you could use `aria-label` is on landmark elements (our [Semantic HTML](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-semantic-html) lesson gets another shoutout... again!):
```